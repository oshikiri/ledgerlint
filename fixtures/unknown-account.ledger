2020-03-26 * super market
  Expences:Unknown expence  600 JPY
  Assets:Cash  -600 JPY

2020-03-26 * super market
  Expences:Food  600 JPY
  Assets:Unknown asset
