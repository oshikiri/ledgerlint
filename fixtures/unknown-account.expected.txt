fixtures/unknown-account.ledger:2 unknown account: Expences:Unknown expence
fixtures/unknown-account.ledger:7 unknown account: Assets:Unknown asset
